libs/utils/utils.js

```diff
@@ -568,3 +568,34 @@ export function createIntersectionObserver({ el, callback, once = true, options
   io.observe(el);
   return io;
 }
+
+const setupSidekickListeners = (() => {
```

Comment:
I'm not sure if utils.js is the best place to put this, but I wanted to have it work for all consumers without them having to explicitly call it.

A consumer will just need to add/update their `tools/sidekick/config.json` with the sendToCaas plugin info and it will work.

Reply:
fyi - refactored so that sidekick code goes into sidekick.js